config/tutorials/otel-logs/vector.toml

```diff
@@ -0,0 +1,45 @@
+[sources.generate_syslog]
+type = "demo_logs"
+format = "syslog"
+count = 100
+
+[transforms.remap_syslog]
+inputs = [ "generate_syslog"]
+type = "remap"
+source = '''
+  structured = parse_syslog!(.message)
+  .timestamp, err = to_unix_timestamp(structured.timestamp)
+  .body = .message
+  del(.message)
+  .resource.source_type = .source_type
+  .resource.host.hostname = structured.hostname
+  .resource.service.name = structured.appname
+  .attributes.syslog.procid = structured.procid
+  .attributes.syslog.facility = structured.facility
+  .attributes.syslog.version = structured.version
+  del(.source_type)
+  .severity = if includes(["emerg", "err", "crit", "alert"], structured.severity) {
+    "ERROR"
+  } else if structured.severity == "warning" {
+    "WARN"
+  } else if structured.severity == "debug" {
+    "DEBUG"
+  } else if includes(["info", "notice"], structured.severity) {
+    "INFO"
+  } else {
+   structured.severity
+  }
+  .name = structured.msgid
+'''
+
+[sinks.emit_syslog]
+inputs = ["remap_syslog"]
+type = "console"
+encoding.codec = "json"
+
+[sinks.quickwit_logs]
+type = "http"
+inputs = ["remap_syslog"]
+encoding.codec = "ndjson"
+uri = "http://host.docker.internal:7280/api/v1/otel-logs/bulk"
```

Comment:
I don't think this `host.docker.internal` feature works seamlessly between platforms and setup

Reply:
I hate having additional commands. This additional argument should do the job "--add-host=host.docker.internal:host-gateway"